### PR TITLE
Add packer configuration for app/OTP/bastion

### PR DESCRIPTION
**Pull request by kshepard**

### CODE REVIEW

File: deployment/packer/cac.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{
+  "variables": {
+    "local_project_directory": ".",
+    "aws_access_key": "",
```
Comment (flibbertigibbet) [inline deployment/packer/cac.json]: Could these be set from the `aws_access_key_id` and `aws_secret_access_key` vars set in the `group_vars` for `cac_secrets`?

**Comment by flibbertigibbet** [comment]
+1

File: deployment/packer/cac.json (hunk 1 above)
Comment (kshepard) [inline deployment/packer/cac.json]: Interesting thought. However, those keys in cac_secrets are actually going to be removed very shortly when we switch to using IAM roles.

Comment (notthatbreezy) [inline deployment/packer/cac.json]: Packer will the common AWS SDK paths (I believe) to grab keys, so passing them in might not be required. We might want to look at some of the authorization helpers in other projects for examples. That way this should be usable with MFA tokens as well, but would require running another command (to authenticate with MFA) before running the packer commands.

Comment (kshepard) [inline deployment/packer/cac.json]: Cool, I do like the idea of using MFA tokens for this process.

**Pull request merged**